dbt-bigquery/src/dbt/adapters/bigquery/catalogs/__init__.py

```diff
@@ -1,4 +1,4 @@
-from dbt.adapters.bigquery.catalogs._generic import (
+from dbt.adapters.bigquery.catalogs._biglake_metastore import (
```

Comment:
similar to question I asked in dbt-bigquery/src/dbt/adapters/bigquery/constants.py. Curious about conventions / choices

Reply:
Are you referring to single underscore versus dunder?

Reply:
why the change in name of the module?

Reply:
we're renaming the catalog type from generic -> biglake_metastore so I figured it makes sense to rename the module as well